frontend/src/store/modules/place/category/actions.js

```diff
@@ -25,5 +25,17 @@ export default {
                     reject(error);
                 });
         });
+    },
+
+    getTagsByCategory: (context, categoryId) => {
+        return new Promise((resolve, reject) => {
+            httpService.get('/places/categories/' + categoryId + '/tags')
```

Comment:
Use `/places/categories/${categoryId}/tags` a bit easier )

Reply:
It was backquotes..
thx